spec/features/projects_manage_spec.rb

```diff
@@ -373,6 +372,12 @@
 
           expect_buttons_to_be_hidden
         end
+
+        it "does not render unlocked button" do
+          visit project_path(id: project.id)
+
```

Comment:
As a personal preference, I often remove these empty lines from small tests like this one. Not a blocker at all, just wanted to share that.